src/lib/Hydra/Plugin/RunCommand.pm

```diff
@@ -38,6 +38,64 @@ sub eventMatches {
     return 0;
 }
 
+sub makeRunCommandJson {
```

Comment:
Technically, this doesn't _actually_  create the json, but the hash that will be encoded into a json. Maybe there's a better name for it (or, move the `encode_json` call in)?

Reply:
Gosh. I don't know what to call it! I don't want to move the encode_json inside it, however it is also not a generic Perl hash: it has JSON-specific values (`true` and `false`) in it.